tools/slang-unit-test/unit-test-cooperative-type-metadata.cpp

```
@@ -0,0 +1,420 @@
+    SlangStage stage,
+    SlangCompileTarget target = SLANG_SPIRV,
+    const char* profileStr = "spirv_1_5")
+{
+    ComPtr<slang::IGlobalSession> globalSession;
+    if (slang_createGlobalSession(SLANG_API_VERSION, globalSession.writeRef()) != SLANG_OK)
+        return {};
+
+    slang::TargetDesc targetDesc = {};
+    targetDesc.format             = target;
+    if (profileStr && target == SLANG_SPIRV)
+        targetDesc.profile = globalSession->findProfile(profileStr);
```

Cppcheck (2.20.0)

[style] The function 'memcpy_s' is never used.

(unusedFunction)